lerna.json

```diff
@@ -1,5 +1,11 @@
 {
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "version": "5.60.0",
   "npmClient": "yarn",
-  "stream": true
+  "stream": true,
+  "command": {
+    "version": {
+      "changelogEntryAdditionalMarkdown": "\nOn our website you can read about our [versioning strategy](https://main--typescript-eslint.netlify.app/users/versioning) and [releases](https://main--typescript-eslint.netlify.app/users/releases)"
```

Comment:
One leading `\n` is enough to start a new paragraph, right?

I wonder why we are using a netlify domain instead of our own?

Also, missing a period, I guess.

Reply:
The custom domain points at the docs that were true for the latest release, but I think we always want to point folks to the freshest copy of the docs (the latest commit on main) when it comes to releases and versioning?

Yes we probably don’t even need the leading newline, just added it to be safe. We can amend it after the first time it is used tomorrow